step.yml

```diff
@@ -24,20 +24,77 @@ toolkit:
   go:
     package_name: github.com/bitrise-steplib/bitrise-step-xcode-start-simulator
 inputs:
-  - destination: platform=iOS Simulator,name=iPhone 8 Plus,OS=latest
-    opts:
-      title: Device destination specifier
-      summary: Destination specifier describes the simulator device to be started.
-      description: |-
-        Destination specifier describes the simulator device to be started.
+- destination: platform=iOS Simulator,name=iPhone 8 Plus,OS=latest
+  opts:
+    title: Device destination specifier
+    summary: Destination specifier describes the simulator device to be started.
+    description: |-
+      Destination specifier describes the simulator device to be started.
 
-        The input value uses the same format as xcodebuild's `-destination` option.
-      is_required: true
+      The input value uses the same format as xcodebuild's `-destination` option.
+    is_required: true
+
+- erase: "no"
+  opts:
+    title: Erase Simulator before use
+    summary: If enabled, will erase a simulator's contents and settings.
+    description: |-
+      If enabled, will erase a simulator's contents and settings.
```

Comment:
Consider adding a reason why someone would want to do this and the impact it would have on build time.